accessly.gemspec

```diff
@@ -21,11 +21,12 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activerecord", ">= 5.0"
+  spec.add_dependency "activerecord", "~> 5.0"
 
   spec.add_development_dependency "bundler", "~> 1.16"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "minitest", "~> 5.0"
   spec.add_development_dependency "database_cleaner"
   spec.add_development_dependency "sqlite3"
+  spec.add_development_dependency "rails", "~> 5.0"
```

Comment:
there's a bug that was recently fixed in rails that supports minitest 5.11.x and generator tests

Reply:
Does this mean we have to depend on rails? Not that it's a big deal, but it would be neat to depend only on activerecord in case anybody wants to use this gem that's using activerecord but not rails.

Reply:
Oooh, I see below that we're using Rails's generator stuff, which means that yes, we need to depend on rails.